pkg/util/p2_exec.go

```diff
@@ -0,0 +1,37 @@
+package util
```

Comment:
Please move to `pkg/exec` or `pkg/p2exec`. I'd like to remove the `util` package altogether.